cf/src/test/scala/typeformation/cf/syntax/InterpolatorsTest.scala

```diff
@@ -44,36 +47,44 @@ class InterpolatorsTest extends FreeSpec with Matchers {
 
       fnJoin"""CREATE DATABASE $getDbName;
 CREATE USER '$DBUsername'@'localhost' IDENTIFIED BY '$DBPassword';""" should ===(
-        FnJoin("", List(
-          "CREATE DATABASE ".asJson,
-          Json.obj("Fn::GetAtt" -> Json.arr("resource".asJson, "dbName".asJson)),
-          ";\nCREATE USER '".asJson,
-          DBUsername.asJson,
-          "'@'localhost' IDENTIFIED BY '".asJson,
-          DBPassword.asJson,
-          "';".asJson)
+        FnJoin(
+          "",
+          List(
+            "CREATE DATABASE ".asJson,
+            Json.obj(
+              "Fn::GetAtt" -> Json.arr("resource".asJson, "dbName".asJson)),
+            ";\nCREATE USER '".asJson,
+            DBUsername.asJson,
+            "'@'localhost' IDENTIFIED BY '".asJson,
+            DBPassword.asJson,
+            "';".asJson
+          )
         )
       )
     }
   }
   "fnSub interpolator" - {
     val param = Parameter.Str("param")
-    implicit val resAttr = ResourceAtt.mk[TestResource]("testAttr")
+    implicit val resAttr =
+      ResourceAtt.mk[TestResource]("testAttr")
 
     "rejects non-token arguments" in {
       illTyped { """fnSub"$param and $resource and ${'notAToken}"""" }
     }
 
     "accepts parameters, resources and pseudoparameters as arguments" in {
-      fnSub"$param, $resource and ${PseudoParameter.Region}" should ===(FnSub("${param}, ${resource} and ${AWS::Region}", None))
+      fnSub"$param, $resource and ${PseudoParameter.Region}" should ===(
```

Comment:
I had overlooked this, but this is really nifty :+1:

Reply:
Is partially a rip-off from [Monsanto's implementation](http://monsantoco.github.io/cloudformation-template-generator/latest/api/com/monsanto/arch/cloudformation/model/Fn$colon$colonJoin.html)